config/values/finance.yml

```diff
@@ -1,19 +1,19 @@
 finance:
```

Comment:
there are a lot build errors on this PR to do with parsing this file - can you help @martyn-w ?